lib/get-sourcemap-content.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const srcURL = require('source-map-url');
+const srcRegExpg = new RegExp(srcURL.regex, 'g');
```

Comment:
i believe `g` flagged regexp are stateful, i suspect it may be unsuitable to save and reuse.